quote-server/server.js

```diff
@@ -3,18 +3,53 @@
 
 //load the 'express' module which makes writing webservers easy
 import express from "express";
+import fetch from "node-fetch";
 //load the quotes JSON
-import quotes from "./quotes.json" assert { type: "json" };
+// import quotes from "./quotes.json" assert { type: "json" };
+let quotes = [];
 
+const fetchQuotes = async () => {
+  try {
+    const quoted = await fetch("https://api.quotable.io/quotes?page=1");
+    const parsedQuote = await quoted.json();
```

Comment:
These variable names are both a little odd and unclear - it's not obvious what `quoted` means, and `parsedQuote` is singular but we expect it to be an array of many quotes - can you think of more clear names?

Reply:
I appreciate your advice. I will modify the names to be more descriptive. Thank you.